README.md

```diff
@@ -2,6 +2,11 @@
 
 **Jump to** [Modal](#modal), [Button](#button)
 
+## To Run
+```
+npm run-script dev-server
```

Comment:
nit: this section should probably go in the contributing section, since the primary use of the readme should be helping people include a component in a project, not creating new ones